resources/projectPortTesting/projectWildfly/pom.xml

```diff
@@ -0,0 +1,207 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.wildfly.quickstarts</groupId>
+        <artifactId>wildfly-quickstart-parent</artifactId>
+        <!--
+        Maintain separation between the artifact id and the version to help prevent
+        merge conflicts between commits changing the GA and those changing the V.
+        -->
+        <version>2</version>
+        <relativePath/>
+    </parent>
+
+    <artifactId>wildfly</artifactId>
+    <version>28.0.0.Beta1-SNAPSHOT</version>
+    <packaging>war</packaging>
+
+    <name>wildfly</name>
+
+    <properties>
+        <!-- The versions for BOMs, Dependencies and Plugins -->
+        <version.microprofile.bom>28.0.0.Beta1</version.microprofile.bom>
+        <version.server.bootable-jar>28.0.0.Beta1</version.server.bootable-jar>
+        <version.wildfly-jar.maven.plugin>8.1.0.Final</version.wildfly-jar.maven.plugin>
```

Comment:
This test project is based on Bootable jar maven plugin that is different from the wildfly-maven-plugin . An example of wildfly-maven-plugin usage can be found there: https://github.com/wildfly/quickstart/blob/main/helloworld/pom.xml#L184

Reply:
I've updated both wildfly and jboss test projects with projects coming from wildfly and JBoss quickstart :)